php/WP_CLI/SearchReplacer.php

```diff
@@ -92,5 +93,13 @@ private function _run( $data, $serialised, $recursion_level = 0, &$visited_data
 
 		return $data;
 	}
+
+	private function str_replace( $from, $to, $data ) {
```

Comment:
On second thought, we probably don't need to make this a private method if it's only used once. Thoughts?